### PR TITLE
Add templates for Riva ASR and TTS

### PR DESCRIPTION
**Pull request by haydeniw**

### CODE REVIEW

File: templates/nvidia_riva/nvidia_riva/chain.py [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    # construct and return the chain
+    return {"user_input": riva_asr} | prompt | model | riva_tts  # type: ignore
+
+
+async def generate_tts_response(mulaw_file: str) -> None:
+    """Process audio with ASR and convert it to text, use the chain
+    to send to the LLM, and send the text from the LLM to Riva TTS.
+    Stores .wav files in ./scratch"""
+
+    # read the audio file
```
Comment (rmkraus) [inline templates/nvidia_riva/nvidia_riva/chain.py:59]: This comment should probably indicate that we are reading the audio file and splitting it into chunks to emulate streaming.

File: templates/nvidia_riva/nvidia_riva/__init__.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
+from nvidia_riva.chain import chain
```
Comment (rmkraus) [inline templates/nvidia_riva/nvidia_riva/__init__.py:1]: Is this import valid? It doesn't look like chain.py has a chain variable.

File: templates/nvidia_riva/nvidia_riva/configuration.py [hunk 3]
```diff
@@ -0,0 +1,125 @@
+"""The definition of the application configuration."""
```
Comment (rmkraus) [inline templates/nvidia_riva/nvidia_riva/configuration.py:1]: I dont think we need this file at all for one of these examples.

**Review by rmkraus — CHANGES_REQUESTED** [review]
Just a few small comments. Also, I'm not sure if we need a tests folder if it is just empty.

**Comment by haydeniw** [comment]
> Also, I'm not sure if we need a tests folder if it is just empty.
@rmkraus unsure why but most or maybe all of of the `templates` have a `tests` folder with an empty `__init__.py`

**Review by rmkraus — APPROVED** [review]
